tests/e2e/Services/Projects/ProjectsConsoleClientTest.php

```diff
@@ -7,6 +7,7 @@
 use Tests\E2E\Scopes\SideClient;
 use Tests\E2E\Services\Projects\ProjectsBase;
 use Tests\E2E\Client;
+use function array_merge;
```

Comment:
```suggestion
```

Not needed, we can use '\' before the function call for a tiny performance increase.